### PR TITLE
fixed issue where bottom of tool-info page was cut off

### PR DESCRIPTION
**Pull request by amenders**

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -136,7 +136,7 @@ a:-webkit-any-link {
   height: calc(100vh - 108px);
 	width: 100vw;
   font-family: "Helvetica Neue";
-  -webkit-overflow-scrolling: touch;
+  overflow-y: scroll;
```
Comment (Jbcampbe) [inline app/styles/app.css]: @amenders I'm pretty sure I was the one who added this. I think it allows for the more natural scrolling on mobile (so it isn't sticky)

Comment (Jbcampbe) [reply app/styles/app.css]: From stack overflow : )

```
-webkit-overflow-scrolling: touch; just tells iOS and other mobile browsers (not sure if there are any more mobile Webkit-based browsers) that the list is scrollable via touch.
```

Comment (Jbcampbe) [reply app/styles/app.css]: But does it solve all of our mobile woes on an actual device?

**Comment by amenders** [comment]
I know that chrome was complaining about the WebKit line, but if somebody wants to test on a mobile device, that would be great. I think even a sticky scroll would be ok for this release.

Amos Endersen
(317) 600-9236

**Pull request merged**